packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -81,9 +85,12 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
                   key={`token${name}`}
                   category={tokenCategory}
                   name={name}
+                  useCamelCase={useJavascriptNames}
                   value={value}
                   comment={comment}
-                  backgroundColor="rgb(255, 255, 255)"
+                  backgroundColor={
+                    tokens['background-colors'].find((token) => token.name === 'color-background-body')?.value
```

Comment:
Optimization: Doing the array find within the tokens map is going to result in extra runs. We can pull this code out of the map to and assign it to a var so the find only happens once.

Reply:
optimizations 😅